### DISCUSSION TITLE
How to set home location view direction?

### DISCUSSION THREAD

**Discussion by andrewbernard** [Stellarium/stellarium#3768] [Q&A] [answered]
I set my location to Melbourne, Australia. But the initial view is always pointing due south. How can one change the view direction for the home location?

**Comment by alex-w** [comment, marked as answer]
![config_dialog_main_tab](https://github.com/Stellarium/stellarium/assets/88731/ab0db8a3-1f68-4828-85cc-1a440535e8c8)

**Reply by andrewbernard** [reply]
How could I have missed that? So dumb. Thanks so much.
